e2e/fixtures/electron-seeded.fixture.ts

```
@@ -0,0 +1,120 @@
+ * - SQL Editor tests
+ * - Connection tests (when you need an existing connection)
+ * - Feature tests that require a project context
+ *
+ * DO NOT use this for:
+ * - Project lifecycle tests (creation, deletion, etc.)
+ * - First-run experience tests
+ * - Tests that need to start from a clean state
+ */
+export const test = base.extend<TestFixtures>({
+  // eslint-disable-next-line no-empty-pattern
+  userData: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)